packages/contracts-bedrock/test/vendor/eas/resolver/SchemaResolver.t.sol

```diff
@@ -0,0 +1,414 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.19;
+
+import { Test } from "forge-std/Test.sol";
+import { SchemaResolver } from "src/vendor/eas/resolver/SchemaResolver.sol";
+import { IEAS, Attestation } from "src/vendor/eas/IEAS.sol";
+import { ISchemaResolver } from "src/vendor/eas/resolver/ISchemaResolver.sol";
+
+// =============================================================
+//                        MOCK CONTRACTS 
+// =============================================================
+
+/// @dev Test implementation of SchemaResolver that always returns true
+contract TestSchemaResolver is SchemaResolver {
+    constructor(IEAS eas) SchemaResolver(eas) {}
+
+    /// @dev Mock attestation validation that always succeeds
+    function onAttest(Attestation calldata, uint256) internal pure override returns (bool) {
+        return true;
+    }
+
+    /// @dev Mock revocation validation that always succeeds
+    function onRevoke(Attestation calldata, uint256) internal pure override returns (bool) {
+        return true;
+    }
+
+    /// @dev Returns fixed version string
+    function version() external pure returns (string memory) {
+        return "1.3.0";
+    }
+}
+
+/// @dev Minimal mock EAS implementation for testing
+contract MockEAS {
```

Comment:
I recommend replacing this `MockEAS` contract with `vm.mockCall`

See https://book.getfoundry.sh/cheatcodes/mock-call?highlight=mockCall#mockcall

You can basically do:

```solidity
vm.mockCall(
  address(eas),
  abi.encodeWithSelector(IEAS.attest.selector),
  abi.encode(true)
)
```

Then when you call `eas.attest`, it will return `true`

This will remove the need to have mock contracts and you can do this `vm.mockCall` inside of `setUp` which will replicate you overwriting `eas` with `MockEAS`